support/Pipelines/Bbh/Inspiral.py

```diff
@@ -89,16 +161,26 @@ def start_inspiral(
     )
 
```

Comment:
Can't reach with github comments, but in this docstring, add that the `ID_INPUT_FILE_PATH` can either be to a SpECTRE ID input file or a SpEC `ID_Params.perl` file